x/twap/listeners.go

```diff
@@ -61,3 +59,27 @@ func (hook *gammhook) AfterJoinPool(ctx sdk.Context, sender sdk.AccAddress, pool
 func (hook *gammhook) AfterExitPool(ctx sdk.Context, sender sdk.AccAddress, poolId uint64, shareInAmount sdk.Int, exitCoins sdk.Coins) {
 	hook.k.trackChangedPool(ctx, poolId)
 }
+
+type concentratedLiquidityListener struct {
+	k Keeper
+}
+
+func (k Keeper) ConcentratedLiquidityListener() concentratedliquiditytypes.ConcentratedLiquidityListener {
+	return &concentratedLiquidityListener{k}
+}
+
+func (l *concentratedLiquidityListener) AfterConcentratedPoolCreated(ctx sdk.Context, sender sdk.AccAddress, poolId uint64) {
+	l.k.mustTrackCreatedPool(ctx, poolId)
```

Comment:
Maybe
```suggestion
	l.k.trackCreatedPool(ctx, poolId)
```

to keep them identical to other methods? (unless there was specific reason why this one has a must)

Reply:
This is intentional. The `trackCreatedPool` requires an error check and a panic. `mustTrackCreatedPool` encapsulates that